libcodechecker/cmd/cmd_line_client.py

```diff
@@ -367,24 +357,29 @@ def printReports(client, reports, output_format):
 
     report_dir_mode = False
     if os.path.isdir(args.newname):
-        # If newname is a valid directory
-        # we assume that it is a report dir
-        # and we are in local compare mode.
-        run_info = check_run_names(client, [args.basename])
+        # If newname is a valid directory we assume that it is a report dir and
+        # we are in local compare mode.
         report_dir_mode = True
     else:
-        run_info = check_run_names(client, [args.basename, args.newname])
-        newid = run_info[args.newname][0]
+        run_info = check_run_names(client, [args.newname])
+        newid = run_info[args.newname].runId
+
+    basename_regex = '^' + args.basename.replace('*', '.*') + '$'
```

Comment:
I think if we only support `*`, in the future it might be more efficient to send this pattern to the SQL server and let it do the pattern matching. Maybe this worth a fixme or a todo comment.